README.md

```diff
@@ -170,6 +171,10 @@ module.exports = {
 
 Jest can be used in projects that use [webpack](https://webpack.js.org/) to manage assets, styles, and compilation. webpack does offer some unique challenges over other tools. Refer to the [webpack guide](https://jestjs.io/docs/en/webpack) to get started.
 
+### Using parcel
+
+Jest can be used in projects that use [parcel-bundler](https://parceljs.org/) to manage assests, styles, and compilation similar to webpack. Parcel bundler offers blazing fast performance utilizing multicore processing, and requires zero configuration.
```

Comment:
```suggestion
Jest can be used in projects that use [parcel-bundler](https://parceljs.org/) to manage assets, styles, and compilation similar to webpack. Parcel requires zero configuration.
```

Reply:
Hey @SimenB ,
Done with the changes suggested, kindly review and merge the **PR**.